src/api/accounts/account-memberships.md

```diff
@@ -22,21 +22,33 @@ some or all of the operations and resources within the account.
 
 ## Models
 
-### Account Membership
+### Account Member
+
+A Member contains extended information about an user's access to an account.
 
 {% h4 Required Fields %}
 
-| Field     | Type               | Description                                                       |
-| :-----    | :-----             | :---------------------------------------------------------------- |
-| accountId | String             | The id of the [Account][] the Membership is scoped to.            |
-| userId    | String             | The id of the user the Account Membership belongs to.             |
-| role      | String             | The role governing Account Membership permissions.                |
-| createdAt | {% dt Timestamp %} | When the Account Membership was created.                          |
+| Field       | Type               | Description                                            |
+| :-----      | :-----             | :------------------------------------------------------|
+| accountId   | String             | The id of the [Account][] the Membership is scoped to. |
+| accountType | String           | The type of the [Account][] the Membership is scoped to.          |
+| accountName | String           | The name of the [Account][] the Membership is scoped to.          |
+| userId      | String             | The id of the user the Membership belongs to.          |
+| role        | String             | The role governing Membership permissions.             |
+
+{% h4 Optional Fields %}
 
+| Field       | Type             | Description                                                       |
+| :-----------| :----------------| :---------------------------------------------------------------- |
+| createdAt  | {% dt Timestamp %} | When the Membership was created.                       |
+| createdBy  | {% dt CRN %}       | Resource that created the  member.                     |
+| modifiedAt | {% dt Timestamp %} | When the Membership was last modified.                 |
+| modifiedBy | {% dt CRN %}       | Resource that last modified the account member.        |
+| version    | {% dt BigNumber %} | Number of times the document has been modified.        |
 
 ## Operations
 
-### Add Account Member **EXPERIMENTAL**
+### Add Member **EXPERIMENTAL**
```

Comment:
I'm just wanting to point out that this endpoint doesn't appear to exist.

Maybe we should remove it and bring it back?